plugin-procfs/README.md

```diff
@@ -1,3 +1,138 @@
+|`kernel_n_procs_blocked`|Gauge|None|Numbers of processes that are blocked on input/output operations|LocalMachine|LocalMachine||
+|`cpu_time_delta`|CounterDiff|NanoSecond|CPU usage|LocalMachine|Process|[kind](#kind)|
+|`memory_usage`|Gauge|Bytes|Memory usage|LocalMachine|Process|[kind](#kind)|
+
+And in case we configure the plugin with the collect of detailed memory metrics :
+
+|Name|Type|Unit|Description|Ressource|RessourceConsumer|Attributes|
+|----|----|----|-----------|---------|-----------------|----------|
+|`mem_total`|Gauge|Bytes|Total amount of physical memory available in the system|LocalMachine|LocalMachine||
+|`mem_free`|Gauge|Bytes|Refers to truly free memory, which is not used by processes, caches or buffers|LocalMachine|LocalMachine||
+|`mem_available`|Gauge|Bytes|More accurate estimate of the actual available memory, taking into account free memory and also caches and buffers that can be freed|LocalMachine|LocalMachine||
+|`cached`|Gauge|Bytes|Cached is the memory used for ROM cache (page cache) in memory|LocalMachine|LocalMachine||
+|`swap_cached`|Gauge|Bytes|Memory that was placed in swap and brought back to RAM, but remains in swap file|LocalMachine|LocalMachine||
+|`active`|Gauge|Bytes|Memory that has been used recently|LocalMachine|LocalMachine||
+|`inactive`|Gauge|Bytes|Used memory that can be reclaimed, freed, or moved to swap.|LocalMachine|LocalMachine||
+|`mapped`|Gauge|Bytes|Memory used for files associated with processes in memory, file and memory mapping.|LocalMachine|LocalMachine||
+
+### Attributes
+
+#### Kind
+
+The kind of the memory is the allocated memory space reserved by the system or the hardware :
+
+|Value|Description|
+|-----|-----------|
+|`resident`|Physical memory actually used by a process|
+|`shared`|Part of memory used simultaneously by several processes|
+|`virtual`|Total virtual memory address space requested by a process|
+
+The kind of the CPU time delta is the average CPU time spent by various tasks :
+
+|Value|Description|
+|-----|-----------|
+|`user`|CPU time used to run userspace processes (not Kernel system processes)|
+|`system`|CPU time used by the system (Kernel)|
+|`guest`|CPU time used by a guest virtual machine when a hypervisor virtualizes environments|
+
+#### cpu_state
+
+The CPU states correspond to all categories of time during which the CPU performs various tasks :
+
+|Value|Description|
+|-----|-----------|
+|`user`|CPU time used to run userspace processes (not Kernel system processes)|
+|`nice`|CPU time consumed by tasks that give way to other higher priority processes|
+|`system`|CPU time used by the system (Kernel)|
+|`idle`|Time the CPU is in sleep mode or waiting for tasks|
+|`irq`|CPU time spent handling hardware interrupts (input/output or peripheral signals)|
+|`softirq`|CPU time spent handling software interrupts (delayed interrupts scheduled by the kernel)|
+|`steal`|Time a hypervisor uses CPU for other virtual machines on the same host|
```

Comment:
some descriptions are in present tense, some are in past tense, please make everything consistent ^^ Use the past everywhere, so `used`, not `uses`.